transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollChannel.java

```diff
@@ -244,6 +247,22 @@ private static boolean isAllowHalfClosure(ChannelConfig config) {
                 ((SocketChannelConfig) config).isAllowHalfClosure();
     }
 
+    boolean interrupted() {
+      if (interrupted) {
+        interrupted = false;
+        return true;
+      }
+      return false;
+    }
+
+    @Override
+    public final void interrupt() {
+        if (!interrupted) {
+          interrupted = true;
+          clearEpollIn();
```

Comment:
This method and its equivalent buddies in the other Channel implementations are either protected or package private methods as they needed to be accessible from the various ChannelConfig implementations. With this change they can be made private.